Tests/MapboxMapsTests/Location/Pucks/Puck2DIntegrationTests.swift

```diff
@@ -1,119 +1,119 @@
-import XCTest
-@testable import MapboxMaps
```

Comment:
This set of tests is broken due to a bad rebase when #359 landed. Suppressing for now. Ticketing out follow up tasks.